src/js/main.js

```diff
@@ -2,47 +2,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function($, Mozilla, Clipboard, dataLayer) {
+
+if (typeof window.dataLayer === 'undefined') {
+    window.dataLayer = [];
+}
+
+(function($, Mozilla, Clipboard, dataLayer, Mzp) {
     'use strict';
 
-    var $episodeLinks = $('.episode-links');
-    var $subscribeLinks = $('.subscribe-links');
-    var $socialShareLinks = $('.social-share-links');
+    var $subscribeButtons = $('.episode-subscribe');
+    var $showNotesLink = $('.episode-shownotes-link');
+    var $subscribeLinks = $('[data-link-type="subscribe"]');
+    var $shareLinks = $('[data-link-type="social share"]');
 
     // set up clipboard.js
-    var copylinkClipboard = new Clipboard('.copylink');
+    var copylinkClipboard = new Clipboard('.js-action-copy a');
 
     // display related copy success message when clicking copy icon
     copylinkClipboard.on('success', function() {
-        $('#modal').find('.copy-ok').addClass('show');
-    });
-
-    // open modals for subscribe/share
-    $episodeLinks.on('click', function(e) {
-        var $target = $(e.target);
-        var modalContentElem;
-        var modalTitle;
-
-        if ($target.hasClass('episode-subscribe-link')) {
-            modalContentElem = $target.data('subscribeLinks');
-            modalTitle = 'Subscribe';
-        } else if ($target.hasClass('episode-share-link')) {
-            modalContentElem = $target.data('shareLinks');
-            modalTitle = 'Share';
-        }
-
-        // mp3 link also exists in this list - that link should have no special handling
-        if (modalContentElem && modalTitle) {
-            e.preventDefault();
-
-            Mozilla.Modal.createModal(e.target, $('#' + modalContentElem), {
-                title: modalTitle
-            });
-        }
+        $('.episode-copy-ok').addClass('show');
+        setTimeout(function() {
+            $('.episode-copy-ok').removeClass('show');
```

Comment:
On season pages, `$(.episode-copy-ok')` returns multiple elements, causing the "URL copied" notice to show in multiple places when clicking the icon.

Reply:
We can't keep this as an Easter Egg? 

Fine, I fixed it 😛